src/components/my-app.js

```diff
@@ -36,7 +36,8 @@ import { menuIcon } from './my-icons.js';
 import './snack-bar.js';
 
 class MyApp extends connect(store)(LitElement) {
-  _render({appTitle, _page, _drawerOpened, _snackbarOpened, _offline}) {
+  render() {
+    const {appTitle, _page, _drawerOpened, _snackbarOpened, _offline} = this;
```

Comment:
Now that `render()` isn't given properties as an argument, I'm still debating whether to use destructure assignment from `this` or just use `this._page` in the template.

Reply:
Personally, I would just use `this.property`. I find it much more straightforward than destructuring `this`.

Reply:
A nice benefit of destructuring though it's that you can see at a glance all the bound properties, which can be nice even if there's not many.